components/InputSpinner/index.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  HStack,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  NumberInputProps,
+  useNumberInput,
+} from '@chakra-ui/react'
+import { AddIcon, MinusIcon } from '@chakra-ui/icons'
+
+interface Props {
+  minusIcon?: React.ReactNode
+  addIcon?: React.ReactNode
+}
+
+export function InputSpinner({ ...props }: Props & NumberInputProps) {
```

Comment:
acho que como vc ja vai desconstruir as props la em baixo nao precisa aqui em cima, certo?

```suggestion
export function InputSpinner(props: Props & NumberInputProps) {
```